src/paperoni/discovery/openreview.py

```diff
@@ -450,8 +452,25 @@ def query(
         block_size: int = 100,
         # Maximum number of results to return
         limit: int = 10000,
+        # A list of focuses
+        focuses: Focuses = None,
     ):
         """Query OpenReview"""
+        if focuses:
+            for focus in focuses.focuses:
+                match focus:
+                    case Focus(drive_discovery=False):
+                        continue
+                    case Focus(type="author_openreview", name=aid, score=score):
```

Comment:
Sorry if you already answered that question but I'm not sure why in openreview only the author_id can be focused (as opposed to openalex for which it is on author name and not the author id). Would both benefits from the ability of focusing on an author name and author id (so something like `author`, `author_openalex` and `author_openreview`)?